src/App/App.js

```diff
@@ -23,20 +23,48 @@ const beValues = [
     caption: 'Card three header',
     content: loremIpsum,
     selected: false
+  }, {
+    id: '3',
+    caption: 'Card four header',
+    content: loremIpsum,
+    selected: false
+  }, {
+    id: '4',
+    caption: 'Card five header',
+    content: loremIpsum,
+    selected: false
+  }, {
+    id: '5',
+    caption: 'Card six header',
+    content: loremIpsum,
+    selected: false
+  }, {
+    id: '6',
+    caption: 'Card seven header',
+    content: loremIpsum,
+    selected: false
+  }, {
+    id: '7',
+    caption: 'Card eight header',
+    content: loremIpsum,
+    selected: false
   }
 ];
 
 
 function App() {
   const [cardsState, setCardsState] = useState(beValues);
+  const [readOnlyState, setReadOnlyState] = useState(false);
 
   const cardsRendered = cardsState.map(card =>
     <Card
       key={card.id}
+      className={'App__card'}
       caption={card.caption}
       content={card.content}
       selected={card.selected}
       editing={card.editing}
```

Comment:
I think it will be better to pass it as one param.
Cos if you need to extern functionality of card you have to correct App.js.
Also I don't see editing field in cards array.

Reply:
There is no 'editing' field in cards array because that array is a simulation of what we get from backend.
Don't think we will ever have such field in the database model.
(And anyway it will be undefined when not initialized.)
Correct me if I didn't get your point